ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,16 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "sinon.js",
+        "sinon-no-sourcemaps.js"
```

Comment:
@priidikvaikla Thank you for your contribution.
Let's modify from A to B. Because we usually use a pattern matcher allowing selection of multiple files. And it's will be readable. Here is what we use [node-glob](https://github.com/isaacs/node-glob#glob-primer).
Thank you.
- A
```js
        "sinon.js",
        "sinon-no-sourcemaps.js"
```

- B
```js
        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Memo: Although there are more release versions from GitHub(78) than npm(72), there are no files can use in front-end, such as `"sinon.js"` , `"sinon-no-sourcemaps.js"` unless use `build.js` to genarate.
So it's fine to use npm auto-update, to get the files which can use in front-end.